File: test_dialogs.js
const fetch = require('node-fetch');

async function test() {
    try {
        const response = await fetch('http://127.0.0.1:8001/api/dialogs');
        const data = await response.json();
        console.log('Диалоги:', data);
        
        if (data.length > 0) {
            const firstDialog = data[0];
            const messagesResponse = await fetch(`http://127.0.0.1:8001/api/dialogs/${firstDialog.telegram_id}/messages`);
            const messages = await messagesResponse.json();
            console.log('Сообщения:', messages);
        }
    } catch (e) {
        console.error('Ошибка:', e);
    }
}

test();

File: test_dialogs.py
import requests
try:
    response = requests.get('http://127.0.0.1:8001/api/dialogs', timeout=10)
    print(f'Диалоги: статус {response.status_code}')
    print(f'Ответ: {response.text}')
except Exception as e:
    print(f'Ошибка диалогов: {e}')
